java/test/org/openqa/selenium/grid/config/JsonConfigTest.java

```diff
@@ -35,15 +37,34 @@ void shouldUseATableAsASection() {
     assertThat(config.get("cheeses", "selected")).isEqualTo(Optional.of("brie"));
   }
 
+  @Test
+  void shouldContainConfigFromArrayOfTables() {
+    String raw = "{\"cheeses\": {\"default\": \"manchego\", "
```

Comment:
Am assuming that this is the JSON that we are working with

```json
{
    "cheeses": {
        "default": "manchego",
        "type": [
            {
                "name": "soft cheese",
                "default": "brie"
            },
            {
                "name": "Medium-hard cheese",
                "default": "Emmental"
            }
        ]
    }
}
```

It would be good if we can add this json format maybe as a javadoc so that the person who is reading the test will be able to understand easily what is the input json that the test is dealing with.


If we were in Java 17 we could have used that `"""my long txt goes here"""` construct :(

Reply:
The "pretty" formatting isn't required, but it's certainly more human-readable. For this unit test, a compact format is used. As indicated, multiline strings aren't supported by Java 11, only introduced in Java 15.

Reply:
@sbabcoc - My suggestion was to include it in the Javadocs so that a person who is reading the test, knows what the input is going to look like. It wasn't necessarily asking for a pretty printed JSON inside the test. The contents of the test is working on a minified format of the JSON, which I understand. But reading a minified JSON that contains a lot of string concatenations, makes it hard for a person to understand the sort of input that is being dealt with here.

Reply:
In my own library, I persist this sort of asset as a resource file, using **Resources.getResource()** to load it. Is this a pattern used in any other part of the Selenium project?

Reply:
```java
    String raw = String.join("", "",
      "{",
        "`cheeses`: {",
          "`default`: `manchego`,",
          "`type`: [",
            "{",
              "`name`: `soft cheese`,",
              "`default`: `brie`",
            "},",
            "{",
              "`name`: `Medium-hard cheese`,",
              "`default`: `Emmental`",
            "}",
          "]",
        "}",
      "}"
    ).replace("`", "\"");
```

Reply:
Still not as clear as un-stringified JSON source, but it's at least slightly more readable. I used the backticks proxies for actual quotes so the text wouldn't be littered with escaped quotes.